internal/monitors/kubernetes/cluster/metadata.yaml

```diff
@@ -141,3 +143,124 @@ monitors:
       on that same dimension.
     dimension: kubernetes_pod_uid
     name: <pod label>
+- <<: *kubernetes
+  monitorType: openshift-cluster
+  sendAll: true
+  dimensions:
+  - description: This is always set to `openshift`
+    name: metric_source
+# NOTE: Do not manually update list below. Update gen-metrics.py and rerun.
+  metrics:
+  - description: Hard limit for number of cpu across all namespaces
```

Comment:
Right now this results in only the openshift metrics instead of the combination of kubernetes and openshift metrics. Apparently [it's not possible](https://github.com/yaml/yaml/issues/35) to merge yaml arrays with anchors and pointers best I can tell. Maybe just link to the kubernetes monitor? Otherwise I think we will have to do some templating or something. Or the gen script can read the kubernetes metrics and print those out with the generated ones. I dunno.

Reply:
Or just make the openshift monitor be specific to OpenShift components and not try and overlap with the kubernetes-cluster monitor?

Reply:
Seems a little janky. It means if you want to use both openshift-cluster and kubernetes-cluster you have to configuration both separately. Could reuse same configuration with yaml anchors but feels clunky.

Reply:
I wouldn't be too concerned about the config because 99% of the time people rely on standard k8s mechanisms (e.g. a service account) to do the authentication, which is the default.